noir/acvm-repo/acir/src/circuit/opcodes/black_box_function_call.rs

```diff
@@ -149,6 +149,11 @@ pub enum BlackBoxFuncCall {
         input: u32,
         outputs: Vec<Witness>,
     },
+    Poseidon2Permutation {
+        inputs: Vec<FunctionInput>,
+        outputs: Vec<Witness>,
+        len: u32,
```

Comment:
can you add a doc comment for what this is?

Reply:
Done